package.json

```diff
@@ -32,8 +32,8 @@
 		"lint:spelling": "cspell \"**\" \".github/**/*\"",
 		"prepare": "husky",
 		"test": "vitest",
-		"test:create": "npx tsx script/create-test-e2e.ts",
-		"test:initialize": "npx tsx script/initialize-test-e2e.ts",
+		"test:create": "tsx script/create-test-e2e.ts",
+		"test:initialize": "tsx script/initialize-test-e2e.ts",
```

Comment:
Unrelated changes?

Reply:
fyi @michaelfaith I removed these changes before merging - but if you think they're good I'm interested in learning more

Reply:
yeah, no worries. That was unrelated. I thought it might have been missed in some repo migration from npm to pnpm.  It seemed strange that everything used pnpm except for these two scripts, which didn't really need `npx` either, since tsx is installed locally.  Unless you're wanting to allow people to run those scripts without needing to install pnpm or any dependencies?  But yeah, it was unrelated to this change; just something I noticed, while testing the changes I made.